pkg/clientaccess/token.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var (
-	defaultClientTimeout = 20 * time.Second
+	defaultClientTimeout = 10 * time.Second
```

Comment:
This brings the join request timeout inline with other internal Kubernetes timeouts.